.docs/user-guide/storage-providers.md

```diff
@@ -718,10 +718,14 @@ mount Google Compute Engine (GCE) persistent disks with GCE machine instances.
 ### Requirements
 
 * GCE account
-* Service account credentials in JSON for GCE project. If not using the Compute
-  Engine default service account, create a new service account with the Service
-  Account Actor role, and create/download a new private key in JSON format. see
-  [creating a service account](https://developers.google.com/identity/protocols/OAuth2ServiceAccount#creatinganaccount)
+* The libStorage server must be running on a GCE instance created with a Service
```

Comment:
Hi @codenrhoden,

@cantbewong suggested the following change:

> The libStorage server must be running on a GCE instance created with a Service Account with appropriate **permissions**, or a Service Account credentials file in JSON format must be supplied. If not using the Compute Engine default Service Account with the Cloud Platform/"all cloud APIs" scope, create a new Service Account via the IAM portal **(Service accounts tab)**. This Service Account requires the **Compute Engine/Instance Admin, Compute Engine/Storage Admin, and Project/Service Account Actor** roles. Then create/download a new private key in JSON format. see creating a service account for details. **Also note that if permissions on a service account are edited, an instance must be restarted to have them take effect.**

I'm suggesting an addendum to Steve's change:

>  The REX-Ray service must be restarted in order for permissions changes to a service account to take effect.